data.md

```diff
@@ -0,0 +1,4 @@
+## Data Retention Policy
```

Comment:
I think this should match index?

```suggestion
## How We Use Data
```

Reply:
We should also add a section about where data is physically hosted, like in https://www.projectsbyif.com/how-if-uses-data/